satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -567,54 +620,54 @@ datasets:
     wavelength: [2.2, 2.25, 2.3]
     file_type: nc_fci_crm
     file_key: mean_clear_sky_reflectance
-    long_name: mean_clear_sky_reflectance_nir22
+    standard_name: toa_bidirectional_reflectance
     vis_channel_id: 7
 
   mean_sza:
     name: mean_sza
     resolution: 1000
     file_type: nc_fci_crm
     file_key: mean_solar_zenith
-    long_name: mean_solar_zenith_angle
+    standard_name: solar_zenith_angle
 
   mean_rel_azi:
     name: mean_rel_azi
     resolution: 1000
     file_type: nc_fci_crm
     file_key: mean_rel_solar_sat_azimuth
-    long_name: mean_relative_solar_satellite_azimuth_angle
+    standard_name: relative_sun_sensor_azimuth_angle
 
   n_acc:
     name: n_acc
     resolution: 1000
     file_type: nc_fci_crm
     file_key: number_of_accumulations
-    long_name: number_of_accumulations
+    standard_name: number_of_accumulations
 
   historical_data:
```

Comment:
`historical_data` is a categorical dataset and we should be able to read `flag_values` and `flag_meanings` information from the enum.

Reply:
Added enum import flag